### PR TITLE
chore: release v1.3.0

### PR DESCRIPTION
**Pull request by eunjae-lee**
Not many things are added since `v1.3.0-beta.0`.
It includes GTM, but it's added as a `chore` commit and it's actually not a feature added to `search-insights`. So I haven't added it to the CHANGELOG.

Last time the PR for `v1.3.0-beta.0`(https://github.com/algolia/search-insights.js/pull/165) was created but I squashed and merged it to `master`.
So at that time only one squashed commit was added to `master`.
I should've just merged it so that all the commits from `develop` are added to `master` as is.

So basically the commits you see below are difference between `v1.2.0` and `v1.3.0`.
This time, I will not squash and just merge it to `master`. Then it'd be easier to compare `develop` and `master`.

image

### CODE REVIEW

File: gtm/src/template.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+const TEMPLATE_VERSION = '1.0.0';
+const INSIGHTS_OBJECT_NAME = 'AlgoliaAnalyticsObject';
+const aa = createArgumentsQueue('aa', 'aa.queue');
+
+function isInitialized() {
+  return !!copyFromWindow(INSIGHTS_OBJECT_NAME);
+}
+
+function formatValueToList(value) {
+  // `20` is the limit that the engine processes.`
```
Comment (francoischalifour) [inline gtm/src/template.js]:
```suggestion
  // `20` is the limit that the engine processes.
```

File: gtm/README.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+The `objectIDs`, `positions` and `queryID` can be forwarded to GTM via [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes).
+
+```js
+instantsearch.widgets.hits({
+  container: '#hits',
+  templates: {
+    item: `
+<article
+ data-insights-objectid="{{objectID}}"
```
Comment (francoischalifour) [inline gtm/README.md]:
```suggestion
 data-insights-object-id="{{objectID}}"
```

File: gtm/README.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+ data-insights-position="{{__position}}"
+ data-insights-query="{{__queryID}}"
```
Comment (francoischalifour) [inline gtm/README.md]:
```suggestion
 data-insights-query-id="{{__queryID}}"
```
